.github/workflows/discovery.yaml

```
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: '0 12 * * TUE'
+  pull_request: # REMOVE BEFORE MERGING
```

```suggestion
  pull_request: # REMOVE BEFORE MERGING
```

Can this be removed?

---

right now I only added it so we can trigger it here on this PR and to make sure it works. But will remove before merging